File: practica01_Rico_Morones_formulageneral.cpp
#include <stdio.h>
#include <math.h>
int main ()
{
int a=10, c=5;
double b=33;
double RR, RP, x;
double ResP, ResN;

printf("dame el valor de a:\n");
scanf("%d", &a);
printf("dame el valor de b:\n");
scanf("%lf", &b);
printf("dame el valor de c:\n");
scanf("%d", &c);

RP= pow (b,2);
x= RP - (4*a*c);
RR= sqrt (x);
ResP= (-b+RR)/(2*a);
ResN= (-b-RR)/(2*a);
printf("Resultado + es: %f\n", ResP);
printf("Resultado - es: %f\n", ResN);
return 0;
}

File: practica01_Rico_Morones_formulageneral.py
from math import sqrt

a=10
b=33
c=5

int(input("Dame el valor de a:"))
int(input("Dame el valor de b:"))
int(input("Dame el valor de c:"))

val1=b*b
val2=val1-4*a*c
val3= sqrt(val2)
val4=-b+val3
val5=-b-val3
val6=2*a
val7=val4/val6
val8=val5/val6
res1=val7
res2=val8
print("x1:", res1)
print("x2:", res2)
